src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/BaseContextMenuStrip.cs

```diff
@@ -197,12 +196,14 @@ private void InitializeContextMenu()
     {
         //this.Opening += new CancelEventHandler(OnContextMenuOpening);
         Name = "designerContextMenuStrip";
+
         if (serviceProvider.GetService(typeof(IUIService)) is IUIService uis)
```

Comment:
```suggestion
        if (serviceProvider.TryGetService(out IUIService? uis)
```
Ditto is a couple places below as well